recipes/pyflann/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: pyflann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    skip: True # [win]
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - flann {{ version }}
+        - cmake
+    run:
+        - python
+        - flann {{ version }}
+        - numpy
+
+test:
+    imports:
+        - pyflann
+
+about:
+    home: http://www.cs.ubc.ca/research/flann/
+    license: BSD 3-Clause
+    license_file: COPYING
+    summary: Python wrapper for FLANN: Fast Library for Approximate Nearest Neighbors
```

Comment:
Same as above. Put in quotes. That or could just reduce to `Python wrapper for FLANN`.